src/covariances.jl

```diff
@@ -0,0 +1,87 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    design_matrix(comps)
+    
+Converts a vector of Composition objects into a Matrix of numbers.
+"""
+function design_matrix(comps)
+    matrix = reduce(hcat, components.(comps)[:,1])'
+    return matrix
+end
+
+"""
+    get_subcomposition_matrix(matrix, entries)
+
+Returns the subcomposition matrix given the `entries` (selected columns) from the `matrix`.
+"""
+function get_subcomposition_matrix(matrix, entries)
+    subcomposition_matrix = matrix[:, entries]
+    return subcomposition_matrix
+end
+
+"""
+    comp_var_array(comps)
+
+Returns the compositional variation array, from definition 4.3 of Aitchson - The Statistical Analysis of Compositional Data.
+"""
+function comp_var_array(comps)
```

Comment:
Please use the names in the issue, short and without underscores.

Reply:
Fixed in commit 88a467ad9064d35fa50fdc68ac48151cf66043a9.